README.md

```diff
@@ -1,5 +1,208 @@
+
+_Sinetiq Framework provides core capabilities that support development and governance methodologies that forms a robust integration ecosystem for small, medium and large enterprise organizations with a heterogenous system landscape._
+
+<p>&nbsp;</p>
+
+## Getting Started
+
+This section will give an in-depth explanation of components used, the `demonstration setup` and hands-on step-by-step `installation guide`.
+
+The demonstration showcase also ships with hands-on code `examples` and components, to run, inspect & modify.
+
+
+### Demonstration Setup
+
+The environment for this demonstration is managed through a client computer and components listed below.
+
+- **API specification registry**: The `Sinetiq API Specification Registry` consist of published and approved api specifications with its generated unique identifier.
+
+- **CLI tool**: The `api-specification-registry-cli` is used to publish approved API specifications to the API specification reqistry.
+
+- **Pipeline examples**: Download and use the `pipeline example` matching your environment, example Azure, Bitbucket, Github and Jenkins. The pipeline examples uses the `CLI tool`, enabling a complete devops (CI/CD) process, to review, approve and publish the api specification to the _API specification repository_.
+
+- **Service Registry** (Consul from Hashicorp): An active service registry for mapping API specification identifiers and endpoints, used to establishing right connections for wanted information exchange.
+
+- **Service Explorer**: List and view approved API specifications, published within the API specification registry, together with active provider API instances, published within the service registry. The `Sinetiq Service Explorer` is a VSCode extension that helps you find service instances, their specifications and verify the implementaition while developing.
+
+<p>&nbsp;</p>
+<div style="text-align:center">
+<img align="center" src="./dosc/media/../../docs/media/api-governance-processes.png" width="600">
+</div>
+  
+<p>&nbsp;</p>
+
+### Installation Guide
+
+#### Registry services
+This will start the registry components, `API specification registry` and `Service Registry`.
+
+1. Fetch the demonstration github repository.   
+    ```sh
+    git clone git@github.com:Sinetiq/api-specification-governance.git
+    ```
+
+2. Change directory to the root git folder where the docker-compose.yml can be found, run the docker command.
+    ```sh
+    cd api-specification-governance
+
+    docker-compose up
+    ```
+
+_You can stop your registry services with CTRL-C._
+
+#### Service explorer
+Download and configure Visual Studio Code as your demonstration editor. Install the SINETIQ Service Explorer plugin. Follow the readme at the visual studio code marketplace [https://marketplace.visualstudio.com/items?itemName=Sinetiq.service-explore](https://marketplace.visualstudio.com/items?itemName=Sinetiq.service-explore).
+
+#### CLI tool: Manual run (not devops pipeline)
```

Comment:
Skip this. There is no need to install the CLI tool. The `npx` command installs any missing libraries.